code_snippets/api-monitor-schedule-downtime.py

```diff
@@ -9,5 +8,15 @@
 
 initialize(**options)
 
+# Repeat for 3 hours (starting now) on every week day for 4 weeks.
```

Comment:
Actually I wrong on my first comment and edited a few minutes after, you must have read the email instead.
The `until_occurrences` means we will reschedule the downtime again 4 times so only 5 days will be muted for 3 hours.
The correct sentence woudl be:
> Downtime for 3 hours (starting now) on every week day and repeat until 4 more recurrences are scheduled